lib/src/main/java/io/fabianterhorst/isometric/IsometricView.java

```diff
@@ -72,6 +74,17 @@ public void setTouchRadius(double touchRadius) {
         this.touchRadius = touchRadius;
     }
 
+    //allow user to set current items with the option to re-measure again or not
+    public void updateItems(List<Isometric.Item> items) {
```

Comment:
The user has the trigger the redrawing of the view manually after this?

Reply:
i will add call to invalidate() to `IsometricView`